national-natural-science-foundation-of-china-2017.csl

```diff
@@ -0,0 +1,206 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only" default-locale="zh-CN">
+  <!-- This style was edited with the Visual CSL Editor (http://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>National Natural Science Foundation of China 2017</title>
+    <title-short>NSFC_2017</title-short>
+    <id>http://www.zotero.org/styles/national-natural-science-foundation-of-china-2017</id>
+    <link href="http://www.zotero.org/styles/national-natural-science-foundation-of-china-2017" rel="self"/>
+    <link href="http://gradschool.ustc.edu.cn/ylb/material/xw/wdxz/19.pdf" rel="documentation"/>
+    <author>
+      <name>Jiongdd</name>
+      <email>wangjiang07@foxmail.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="engineering"/>
+    <category field="science"/>
+    <category field="generic-base"/>
+    <summary>This style just partly implemented what the National Natural Science Foundation of China requires.</summary>
+    <updated>2017-03-08T03:24:28+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author">
+      <name delimiter-precedes-last="always" et-al-min="999" et-al-use-first="999" initialize-with=" " name-as-sort-order="all" sort-separator=" ">
```

Comment:
@jiongdd, "et-al-use-first" (the number of authors to show before "et al." [and others]) must always have a lower value than "et-al-min" (the number of authors beyond which name lists are abbreviated).